addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -363,27 +363,27 @@ export function applyInlineStyle(editor, applyStyle, style, shouldApply=true) {
 }
```

Comment:
About the commit message: I'm not sure the `?` at the end of the first sentence of the second paragraph is intentional ? If it is then I don't undersand the sentence... ^^' Also, I'd remove the capital letter of "Fix" in the last paragraph.